src/server/db/mongo.js

```diff
@@ -8,6 +8,54 @@
 
 const LRU = require('lru-cache');
 
+/**
+ * InMemory data storage.
```

Comment:
Why is there is a copy of `InMemory` from `src/server/db/inmemory.js` here?

Reply:
my mistake, this file should not have changed - fix coming up in the next commit